lib/typings.d.ts

```diff
@@ -34,15 +34,17 @@ export type CommitteeCheckError =
 | 'MAJORITY_NOT_FOUND'
 | 'MINORITY_RESULT'
 
+export type ConsensusEvaluation =
+  | 'MAJORITY_RESULT'
+  | CommitteeCheckError
```

Comment:
should this be called `ConsensusCheckError`?

Reply:
Done in 85c882d

Reply:
I decided to use `ConsensusNotFoundReason`, see 3680486